### PR TITLE
build: moving health checkers to extensions

### PR DESCRIPTION
**Pull request by alyssawilk**
Risk Level: medium (folks with custom builds must update)
Testing: existing tests
Docs Changes: n/a
Release Notes: inline

### CODE REVIEW

**Comment by alyssawilk** [comment]
cc @Augustyniak I assume removing health checkers from the E-M build is OK for y'all?

**Comment by alyssawilk** [comment]
nope - h2 keepalive is a pure h2 thing so should be fine.

**Comment by Augustyniak** [comment]
🟢 green light from Lyft then 🟢

File: tools/code_format/check_format.py [hunk 1]
```diff
@@ -1062,7 +1062,7 @@ def check_visibility(error_messages):
             if output:
                 error_messages.append(
                     "This change appears to add visibility rules. Please get senior maintainer "
-                    "approval to add an exemption to check_visibility tools/code_format/check_format.py"
+                    "approval to add an exemption to visibility_excludes in tools/code_format/config.yaml"
```
Comment (RyanTheOptimist) [inline tools/code_format/check_format.py:1065]: Thanks for doing this!!

**Review by RyanTheOptimist — DISMISSED** [review]
Awesome! Thanks for doing this!!

One nit.

**Comment by alyssawilk** [comment]
@snowp for non-google sign-off

**Review by snowp — APPROVED** [review]
Nice, LGTM!

**Review by RyanTheOptimist — DISMISSED** [review]

**Review by RyanTheOptimist — APPROVED** [review]

**Pull request merged**